dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -90,6 +91,26 @@ UINT MapVerbToDfmCmd(_In_ LPCSTR verba)
     return 0;
 }
 
+static bool IsVerbListSeparator(int Ch)
+{
+    return Ch == ' ' || Ch == ','; // learn.microsoft.com/en-us/windows/win32/shell/context-menu-handlers
+}
+
+static int FindVerbInDefaultVerbList(LPCWSTR List, LPCWSTR Verb)
```

Comment:
```suggestion
static INT FindVerbInDefaultVerbList(LPCWSTR List, LPCWSTR Verb)
```

Reply:
Side note: I always cringe inside when I see functions that return signed integers for indices

Reply:
How is `INT` better than `int`?

> Side note: I always cringe inside when I see functions that return signed integers for indices

Whatever, how many lists have more than 2 billion items? This one is limited to 255 / 2 at most (and that will never happen, no verbs are actually a single character and certainly not 130 of them).